cache/clock_cache.h

```diff
@@ -295,28 +562,37 @@ class ClockHandleTable {
  private:
   friend class ClockCacheShard;
 
-  int FindVisibleElement(const Slice& key, int& probe, int displacement);
+  int FindElement(const Slice& key, uint32_t hash, int& probe);
```

Comment:
1. int& probe --> int probe
2. probe is int or uint32_t type?

Reply:
1. and 3. We need a reference because it's our way to keep track of progress across consecutive calls to these find functions, so the i-th call can take over where the (i-1)-th left. (We update probe in FindSlot.)
2. Unsigned. Will fix.